### DISCUSSION TITLE
Fuzzing python code

### DISCUSSION THREAD

**Discussion by rubenrdpuma** [google/oss-fuzz-gen#76] [Q&A] [answered]
I just came across this project and I was wondering whether it is possible to use it against python code.

From the project description I read "This framework generates fuzz targets for real-world C/C++ projects" but on the other hand the project is based on OSS-Fuzz, which "supports C/C++, Rust, Go, Python and Java/JVM code. Other languages supported by [LLVM](https://llvm.org/) may work too." 

Then, is OSS-Fuzz-Gen then suitable only for C/C++ or may it also be used against Python scripts? In that case, how should I configure the framework to do so? Otherwise, what's missing to make it possible to fuzz Python code if OSS-Fuzz already supports python?

Thanks and sorry if this is a silly question.

**Comment by oliverchang** [comment, marked as answer]
Hi! 

Thanks for the question, and sorry for the slow reply. 

In theory we could probably support Python fairly easily, given the FuzzIntrospector already supports Python. Our prompts would likely also need to be slightly rewritten as they are currently very C/C++ centric.

@DavidKorczynski @DonggeLiu thoughts?

**Reply by DonggeLiu** [reply]
Yep, that's a good idea.
I suppose we can do this after #62?
We can keep this in mind while refactoring the code.
